rfcs/text/0033/io.yml

```diff
@@ -0,0 +1,43 @@
+- name: io
+  title: IO
+  group: 2
+  short: Fields describing a batch of input or output data produced by a process.
+  description: >
+    Fields describing a batch of input or output (IO) data produced by a process.
+    These fields should only appear on the top level process object when event.action is 'io'
+  type: group
+  reusable:
```

Comment:
If the only foreseeable use for these fields is at `process.*`, what about adding directly under `process.*` instead of as a reuse?